### PR TITLE
Fix two sysvars' default values.

### PR DESCRIPTION
**Pull request by CbcWestwolf**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Change the default values of these system variables:
* `tidb_enable_mutation_checker`: `OFF` -> `ON`
* `tidb_txn_assertion_level`: `OFF` -> `FAST`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -782,7 +782,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`OFF`
+- 默认值：`ON`
 - 这个变量用于设置是否开启 mutation checker。mutation checker 是一项在 DML 语句执行过程中进行的数据索引一致性校验，校验报错会回滚当前语句。开启该校验会导致 CPU 使用轻微上升。详见[数据索引一致性报错](/troubleshoot-data-inconsistency-errors.md)。
```
Comment (ekexium) [inline system-variables.md:787]:
```suggestion
- 对于新创建的 v6.0 及以上的集群，默认值为 `ON`。对于升级版本的集群，如果升级前是 v6.0 以下版本，升级后默认值为 `OFF`。

```

File: system-variables.md [hunk 2]
```diff
@@ -1866,7 +1866,7 @@ set tidb_slow_log_threshold = 200;
 
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`OFF`
+- 默认值：`FAST`
 - 可选值：`OFF`，`FAST`，`STRICT`
 - 这个变量用于设置 assertion 级别。assertion 是一项在事务提交过程中进行的数据索引一致性校验，它对正在写入的 key 是否存在进行检查。如果不符则说明数据索引不一致，会导致事务 abort。详见[数据索引一致性报错](/troubleshoot-data-inconsistency-errors.md)。
```
Comment (ekexium) [inline system-variables.md:1872]:
```suggestion
- 对于新创建的 v6.0 及以上的集群，默认值为 `FAST`。对于升级版本的集群，如果升级前是 v6.0 以下版本，升级后默认值为 `OFF`。

```

**Comment by TomShawn** [comment]
@CbcWestwolf 这些默认值变更是在 v6.0.0 就已经发生的呢？

**Comment by CbcWestwolf** [comment]
> @CbcWestwolf 这些默认值变更是在 v6.0.0 就已经发生的呢？

是的，新集群和升级上来的集群的默认值是不一样的： https://github.com/pingcap/docs-cn/commit/8c5c352ecd6df3a10aa7c3009fa3a10f0795ea2d#r75059856

**Comment by ran-huang** [comment]
@CbcWestwolf 麻烦看下 release notes 的修改 [ea3de88](https://github.com/pingcap/docs-cn/pull/9660/commits/ea3de8877d4826b27001d4182853a716056dc26c)

**Comment by CbcWestwolf** [comment]
> @CbcWestwolf 麻烦看下 release notes 的修改 [ea3de88](https://github.com/pingcap/docs-cn/pull/9660/commits/ea3de8877d4826b27001d4182853a716056dc26c)

Thanks! LGTM.

File: system-variables.md [hunk 3]
```diff
@@ -782,8 +782,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`OFF`
+- 默认值：`ON`
 - 这个变量用于设置是否开启 mutation checker。mutation checker 是一项在 DML 语句执行过程中进行的数据索引一致性校验，校验报错会回滚当前语句。开启该校验会导致 CPU 使用轻微上升。详见[数据索引一致性报错](/troubleshoot-data-inconsistency-errors.md)。
+- 对于新创建的 v6.0 及以上的集群，默认值为 `ON`。对于升级版本的集群，如果升级前是低于 v6.0 的版本，升级后默认值为 `OFF`。
```
Comment (ran-huang) [inline system-variables.md:787]:
```suggestion
- 对于新创建的 v6.0.0 及以上的集群，默认值为 `ON`。对于升级版本的集群，如果升级前是低于 v6.0.0 的版本，升级后默认值为 `OFF`。
```

File: system-variables.md [hunk 4]
```diff
@@ -1866,9 +1867,10 @@ set tidb_slow_log_threshold = 200;
 
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`OFF`
+- 默认值：`FAST`
 - 可选值：`OFF`，`FAST`，`STRICT`
 - 这个变量用于设置 assertion 级别。assertion 是一项在事务提交过程中进行的数据索引一致性校验，它对正在写入的 key 是否存在进行检查。如果不符则说明数据索引不一致，会导致事务 abort。详见[数据索引一致性报错](/troubleshoot-data-inconsistency-errors.md)。
+- 对于新创建的 v6.0 及以上的集群，默认值为 `FAST`。对于升级版本的集群，如果升级前是低于 v6.0 的版本，升级后默认值为 `OFF`。
```
Comment (ran-huang) [inline system-variables.md:1873]:
```suggestion
- 对于新创建的 v6.0.0 及以上的集群，默认值为 `FAST`。对于升级版本的集群，如果升级前是低于 v6.0.0 的版本，升级后默认值为 `OFF`。
```

**Review by ran-huang — APPROVED** [review]

**Review by ekexium — APPROVED** [review]
LGTM

**Pull request merged**